docs/pages/api-reference/sources/redshift.md

```diff
@@ -1,53 +1,38 @@
 ---
 title: Redshift
 order: 0
-status: draft
+status: published
 ---
 ### Redshift
-Data connector to Snowflake databases.
+Data connector to Redshift databases.
 
 #### Database Parameters
 <Expandable title="name" type="str">
 A name to identify the source. The name should be unique across all Fennel sources.
 </Expandable>
 
-<Expandable title="account" type="str">
-
-Snowflake account identifier. This is the first part of the URL used to access 
-Snowflake. For example, if the URL is `https://<account>.snowflakecomputing.com`, 
-then the account is `<account>`. 
-
-This is usually of the form `<ORG_ID>-<ACCOUNT_ID>`. Refer to the 
-[Snowflake documentation](https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-organization-and-account-name-for-an-account) 
-to find the account identifier.
+<Expandable title="s3_access_role_arn" type="str">
```

Comment:
I think we need to provide some more information around this. 

Potentially provide some IAM policy JSON, trust relationships etc to make our customer's experience smoother.

Reply:
Got it. Will check with @nikhilgarg28 on this and update the documentation

Reply:
Updated the documentation with steps to add s3_access_role_arn. Added sample policy JSON as well